easybuild/easyconfigs/h/HOMER/HOMER-4.11-foss-2023a-R-4.3.2.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'Binary'
+
+name = 'HOMER'
+version = '4.11'
+versionsuffix = '-R-%(rver)s'
+
+homepage = "http://homer.ucsd.edu/homer/"
+description = """HOMER (Hypergeometric Optimization of Motif EnRichment) is a suite of tools for Motif Discovery and
+ next-gen sequencing analysis.  It is a collection of command line programs for unix-style operating systems written
+ in Perl and C++. HOMER was primarily written as a de novo motif discovery algorithm and is well suited for finding
+ 8-20 bp motifs in large scale genomics data.  HOMER contains many useful tools for analyzing ChIP-Seq, GRO-Seq,
+ RNA-Seq, DNase-Seq, Hi-C and numerous other types of functional genomics sequencing data sets."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+source_urls = ['http://homer.ucsd.edu/homer']
+sources = ['configureHomer.pl']
+checksums = ['ccdaa3004a0e0df0882634671d4a1acc88364761e0e6c7ea329ebbf1eb729537']
+
+dependencies = [
+    ('Perl', '5.36.1'),
+    ('R', '4.3.2'),
+    ('SAMtools', '1.18'),
+    ('R-bundle-Bioconductor', '3.18', versionsuffix)
+]
+
+postinstallcmds = ["cd %(installdir)s && perl ./configureHomer.pl -install -all"]
```

Comment:
Make sure the specific version is installed, as is done in https://github.com/bear-rsg/easybuild-easyconfigs/blob/2020a/easybuild/easyconfigs/h/HOMER/HOMER-4.11-foss-2020a.eb